testdata/expected/python-link-tests/python-link-tests/PythonTestPackage.graph.json

```diff
@@ -162,8 +195,18 @@
       "Unit": "python-link-tests",
       "Def": true,
       "File": "pkg0/tests/test_m0.py",
-      "Start": 82,
-      "End": 85
+      "Start": 92,
+      "End": 95
+    },
+    {
+      "DefUnitType": "PythonTestPackage",
+      "DefUnit": "python-link-tests",
+      "DefPath": "test_m0.py/test_m0.bar.hello",
```

Comment:
Why does it not resolve this reference correctly? I would think that if it is able to resolve references to the module `m0`, it should also be able to resolve references to the module `m1`?